frontend/src/authConfig.ts

```diff
@@ -0,0 +1,73 @@
+import { Configuration, LogLevel } from '@azure/msal-browser';
+
+export const b2cPolicies = {
+    names: {
+        signUpSignIn: 'B2C_1_signupsignin',
+        // forgotPassword: 'B2C_1_reset_v3',
+        // editProfile: 'B2C_1_edit_profile_v2',
+    },
+    authorities: {
+        signUpSignIn: {
+           authority: 'https://stockhubb2c.b2clogin.com/stockhubb2c.onmicrosoft.com/B2C_1_signupsignin',
+            //authority: "https://stockhubb2c.b2clogin.com/tfp/2b9bbbf5-4706-4eed-86a7-d55a5094b647/B2C_1_signupsignin",
+        },
+        // forgotPassword: {
+        //     authority: 'https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/B2C_1_reset_v3',
+        // },
+        // editProfile: {
+        //     authority: 'https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/b2c_1_edit_profile_v2',
+        // },
```

Comment:
issue(blocking): commented code to remove

Reply:
je garde car après on va mettre le profil